test/functional/basic/basic-e2e-specs.js

```diff
@@ -202,8 +203,12 @@ describe('XCUITestDriver - basics', function () {
     });
 
     it('should start a session, navigate to url, get title', async () => {
-      let contexts = await driver.contexts();
-      contexts.length.should.be.at.least(2);
+      let contexts;
+      await retryInterval(10, 1000, async () => {
+        // on some systems (like Travis) it takes a while to load the webview
+        contexts = await driver.contexts();
+        contexts.length.should.be.at.least(2);
```

Comment:
Getting contexts should not fail, unless something is seriously wrong. But if only one context is there the assertion will throw an error.